package/kernel/linux/modules/hwmon.mk

```diff
@@ -78,6 +78,21 @@ endef
 $(eval $(call KernelPackage,hwmon-adt7475))
 
 
+define KernelPackage/hwmon-coretemp
+  TITLE:=Pentium thermal monitoring support
+  KCONFIG:=CONFIG_SENSORS_CORETEMP
+  FILES:=$(LINUX_DIR)/drivers/hwmon/coretemp.ko
+  AUTOLOAD:=$(call AutoProbe,coretemp)
+  $(call AddDepends/hwmon)
+endef
```

Comment:
This is at least activated by default in target/linux/x86/64/config-default 
Should this also be activated by default for more x86 configurations?

Reply:
> Should this also be activated by default for more x86 configurations?

@hauke We could, but then (a) `hwmon` would need to be baked into the kernel, and do we do it for all sensors, i.e. `CORETEMP`, `K10TEMP`, `K8TEMP`, `VIA_CPUTEMP`?

Reply:
Yes